python/scripts/manifest_sample.csv

```diff
@@ -1,4 +1,5 @@
-include_flag,destination_table,unique_data_id,data_date,local_folder,s3_folder,filepath,notes
-use,project_temp,project_sample,4/1/2016,../../../data,https://s3.amazonaws.com/housinginsights/,sample/project_sample.csv,Sample data
-use,acs_rent_median_temp,acs_sample,7/1/2012,../../../data,https://s3.amazonaws.com/housinginsights/,sample/acs_median_rent_sample.csv,Sample data
-use,acs_rent_median_temp,acs_rent_median_09_5YR,7/1/2007,../../../data,https://s3.amazonaws.com/housinginsights/,raw/acs/B25058_median_rent_by_tract/2009_5year/ACS_09_5YR_B25058_with_ann.csv,Ready to go data at the census tract level. Do pay attention to margins of error
+include_flag,destination_table,unique_data_id,data_date,local_folder,s3_folder,filepath,notes
+use,project_temp,project_sample,4/1/16,../../../data,https://s3.amazonaws.com/housinginsights/,sample/project_sample.csv,Sample data
+use,acs_rent_median_temp,acs_sample,7/1/12,../../../data,https://s3.amazonaws.com/housinginsights/,sample/acs_median_rent_sample.csv,Sample data
+use,acs_rent_median_temp,acs_rent_median_09_5YR,7/1/07,../../../data,https://s3.amazonaws.com/housinginsights/,raw/acs/B25058_median_rent_by_tract/2009_5year/ACS_09_5YR_B25058_with_ann.csv,Ready to go data at the census tract level. Do pay attention to margins of error
+use,reac_scores,reac_scores_20170315,3/15/17,../../../data,https://s3.amazonaws.com/housinginsights/,raw/preservation_catalog/20170315/Reac_score.csv,
```

Comment:
Let's pull this line out of the sample for the purposes of the dev version (it's a good place to user for testing, but should clean up afterwards)